src/main/java/org/opensearch/security/dlic/rest/api/RestApiAdminPrivilegesEvaluator.java

```diff
@@ -159,6 +153,9 @@ public boolean isCurrentUserAdminFor(final Endpoint endpoint) {
     }
 
     private static String buildEndpointActionPermission(final Endpoint endpoint, final String action) {
+        if (action == null) {
```

Comment:
Why add the null check?  I'd rather see Object.requreNonNull(...), I'd consider this a coding error, not a runtime state, no?

Reply:
Addressed